tests/apps/actorstate/app.go

```diff
@@ -227,6 +232,7 @@ func appRouter() http.Handler {
 
 	router.HandleFunc("/", indexHandler).Methods("GET")
 	router.HandleFunc("/dapr/config", configHandler).Methods("GET")
+	router.HandleFunc("/healthz", healthzHandler).Methods("GET")
```

Comment:
This was missing in actorstate app and was causing test failures.  Implementing /healthz is required by the actors contract.